install-dev/langs/en/data/tab.xml

```diff
@@ -108,7 +108,7 @@
     <tab id="Stock_Coverage" name="Stock Coverage" />
     <tab id="Stock_Management" name="Stock Management" />
     <tab id="Stock_Movement" name="Stock Movement" />
-    <tab id="Stock" name="Stock" />
+    <tab id="Stock" name="Stocks" />
```

Comment:
Again, I wonder if the plural is necessary here.
In that context, it seems to me that "Stock" is uncountable.